src/main/java/org/beanpod/switchboard/controller/StreamLogController.java

```diff
@@ -0,0 +1,21 @@
+package org.beanpod.switchboard.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.beanpod.switchboard.dao.StreamLogDaoImpl;
+import org.openapitools.api.StreamlogApi;
+import org.openapitools.model.StreamLogModel;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class StreamLogController implements StreamlogApi {
+  private final StreamLogDaoImpl streamLogDao;
+
+  @Override
+  public ResponseEntity<List<StreamLogModel>> retrieveStreamLogs(@PathVariable Long streamId) {
+    return ResponseEntity.ok(streamLogDao.getStreamLogs(streamId));
+  }
+}
```

Comment:
why not make the stream logs in the logs controller

Reply:
also maybe the endpoint should start with /log to  be consistent with the log controller

Reply:
i was going to do it this way, but /log/{XYZ} is already taken by another log endpoint. I can do something like /log/stream/{streamID}, what do you think?

Reply:
yeah i was thinking /log/stream/{streamID}

Reply:
I agree with @mohhef

Reply:
the endpoint has been updated to /log/stream/{streamID}